mysql/metadata.csv

```diff
@@ -42,3 +42,108 @@ mysql.replication.seconds_behind_master,gauge,,second,,The lag in seconds betwee
+mysql.innodb.log_waits,gauge,,wait,second,"The number of times that the log buffer was too small and a wait was required for it to be flushed before continuing.",0,mysql,mysql innodb log_waits
+mysql.innodb.log_write_requests,gauge,,write,second,"The number of write requests for the InnoDB redo log.",0,mysql,mysql innodb log_write_requests
+mysql.innodb.log_writes,gauge,,write,second,"The number of physical writes to the InnoDB redo log file.",0,mysql,mysql innodb log_writes
+mysql.innodb.lsn_current,gauge,,,,"Log sequence number as shown in the LOG section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb lsn_current
+mysql.innodb.lsn_flushed,gauge,,,,"Flushed up to log sequence number as shown in the LOG section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb lsn_flushed
+mysql.innodb.lsn_last_checkpoint,gauge,,,,"Log sequence number last checkpoint as shown in the LOG section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb lsn_last_checkpoint
+mysql.innodb.mem_adaptive_hash,gauge,,,,"As shown in the BUFFER POOL AND MEMORY section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb mem_adaptive_hash
+mysql.innodb.mem_dictionary,gauge,,,,"As shown in the BUFFER POOL AND MEMORY section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb mem_dictionary
+mysql.innodb.mem_total,gauge,,,,"As shown in the BUFFER POOL AND MEMORY section of the SHOW ENGINE INNODB STATUS output.",0,mysql,mysql innodb mem_total
+mysql.innodb.os_log_pending_fsyncs,gauge,,operation,,"Number of pending InnoDB log fsync (sync-to-disk) requests.",0,mysql,mysql innodb os_log_pending_fsyncs
+mysql.innodb.os_log_pending_writes,gauge,,write,,"Number of pending InnoDB log writes.",0,mysql,mysql innodb os_log_pending_writes
+mysql.innodb.os_log_written,gauge,,byte,second,"Number of bytes written to the InnoDB log.",0,mysql,mysql innodb os_log_written
+mysql.innodb.pages_created,gauge,,page,second,Number of InnoDB pages created.,0,mysql,mysql innodb pages_created
+mysql.innodb.pages_read,gauge,,page,second,Number of InnoDB pages read.,0,mysql,mysql innodb pages_read
+mysql.innodb.pages_written,gauge,,page,second,Number of InnoDB pages written.,0,mysql,mysql innodb pages_written
+mysql.innodb.rows_deleted,gauge,,row,second,Number of rows deleted from InnoDB tables.,0,mysql,mysql innodb rows_deleted
+mysql.innodb.rows_inserted,gauge,,row,second,Number of rows inserted into InnoDB tables.,0,mysql,mysql innodb rows_inserted
+mysql.innodb.rows_read,gauge,,row,second,Number of rows read from InnoDB tables.,0,mysql,mysql innodb rows_read
+mysql.innodb.rows_updated,gauge,,row,second,Number of rows updated in InnoDB tables.,0,mysql,mysql innodb rows_updated
+mysql.innodb.s_lock_os_waits,gauge,,,second,"As shown in the SEMAPHORES section of the SHOW ENGINE INNODB STATUS output",0,mysql,mysql innodb s_lock_os_waits
```

Comment:
Those desc are the same, maybe we can make them more specific.

Reply:
I'm not sure what all those metric mean so it's going to be hard to be more specific. They all come from InnoDB status, but the output is not clearly documented:
https://mariadb.com/kb/en/show-engine-innodb-status/
https://dev.mysql.com/doc/refman/8.0/en/innodb-standard-monitor.html